src/constbuffer.c

```diff
@@ -183,6 +183,42 @@ IMPLEMENT_MOCKABLE_FUNCTION(, CONSTBUFFER_HANDLE, CONSTBUFFER_CreateWithMoveMemo
     return (CONSTBUFFER_HANDLE)result;
 }
 
+IMPLEMENT_MOCKABLE_FUNCTION(, CONSTBUFFER_HANDLE, CONSTBUFFER_CreateWithMoveMemoryWithCustomFree, unsigned char*, source, uint32_t, size, CONSTBUFFER_CUSTOM_FREE_FUNC, customFreeFunc, void*, customFreeFuncContext)
```

Comment:
>CONSTBUFFER_CreateWithMoveMemoryWithCustomFree [](http://example.com/codeflow?start=50&length=46)

This function looks identical to CONSTBUFFER_CreateWithCustomFree, except for a missing `const`  and a removed NULL check.  I'm not sure how this is different or why we need this.